# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: ddm\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2013-10-03 09:21+0200\n"
"PO-Revision-Date: 2013-12-28 11:43+0100\n"
"Last-Translator: Schoelje <schoelje@solydxk.com>\n"
"Language-Team: SolydXK <admin@solydxk.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 1.5.4\n"
"Language: cs\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Poedit-Basepath: .\n"
"X-Poedit-SearchPath-0: /home/toor/DATA\n"

#: usr/lib/ddm/ati.py:72
#, python-format
msgid "ATI card found: %(card)s"
msgstr "Nalezena karta ATI: %(card)s"

#: usr/lib/ddm/ati.py:77
#, python-format
msgid "ATI chip serie found: %(serie)s"
msgstr "Nalezen čip ATI série: %(serie)s"

#: usr/lib/ddm/ati.py:86
#, python-format
msgid "ATI driver found: %(drv)s (%(status)s)"
msgstr "Nalezen ovladač ATI: %(drv)s (%(status)s)"

#: usr/lib/ddm/ati.py:89 usr/lib/ddm/intel.py:56 usr/lib/ddm/nvidia.py:104
#: usr/lib/ddm/via.py:55
#, python-format
msgid "Driver not installable: %(drv)s"
msgstr "Ovladač: %(drv)s nelze nainstalovat"

#: usr/lib/ddm/ati.py:91
#, python-format
msgid "ATI chip serie not supported: %(serie)d"
msgstr "Čip ATI série: %(serie)d není podporován"

#: usr/lib/ddm/ati.py:93
#, python-format
msgid "No driver found for: %(card)s (%(man)s:%(serie)s)"
msgstr "Pro: %(card)s (%(man)s:%(serie)s) nenalezen ovladač"

#: usr/lib/ddm/ati.py:106 usr/lib/ddm/intel.py:68 usr/lib/ddm/nvidia.py:118
#: usr/lib/ddm/via.py:67
#, python-format
msgid "Install driver: %(drv)s"
msgstr "Instalovat ovladač: %(drv)s"

#: usr/lib/ddm/ati.py:120 usr/lib/ddm/nvidia.py:137
#, python-format
msgid "Switch to module: %(module)s"
msgstr "Přepnout na modul: %(module)s"

#: usr/lib/ddm/ati.py:126 usr/lib/ddm/intel.py:77 usr/lib/ddm/nvidia.py:147
#: usr/lib/ddm/via.py:76
#, python-format
msgid "Driver installed: %(drv)s"
msgstr "Nainstalován ovladač: %(drv)s"

#: usr/lib/ddm/ati.py:137
msgid "Removing ATI drivers"
msgstr "Odebrání ovladačů ATI"

#: usr/lib/ddm/ati.py:151 usr/lib/ddm/intel.py:96 usr/lib/ddm/nvidia.py:176
#: usr/lib/ddm/via.py:95
#, python-format
msgid "Driver removed: %(drv)s"
msgstr "Odebrán ovladač: %(drv)s"

#: usr/lib/ddm/ati.py:159
msgid "ATI display driver"
msgstr "ATI ovladač displeje"

#: usr/lib/ddm/ati.py:161 usr/lib/ddm/intel.py:104 usr/lib/ddm/nvidia.py:186
msgid "Intel display driver"
msgstr "Intel ovladač displeje"

#: usr/lib/ddm/ati.py:163
msgid "Radeon display driver"
msgstr "Radeon ovladač displeje"

#: usr/lib/ddm/ati.py:165 usr/lib/ddm/intel.py:106 usr/lib/ddm/nvidia.py:190
#: usr/lib/ddm/via.py:105
msgid "Framebuffer display driver"
msgstr "Framebuffer ovladač displeje"

#: usr/lib/ddm/ati.py:167 usr/lib/ddm/intel.py:108 usr/lib/ddm/nvidia.py:192
#: usr/lib/ddm/via.py:107
msgid "Vesa display driver"
msgstr "Vesa ovladač displeje"

#: usr/lib/ddm/ati.py:230
#, python-format
msgid "Install aborted: not in repository: %(repo)s"
msgstr "Instalace ukončena: v repozitáři chybí balíček: %(repo)s"

#: usr/lib/ddm/broadcom.py:88
#, python-format
msgid "Broadcom chip serie found: %(chip)s"
msgstr "Nalezen čip série Broadcom: %(chip)s"

#: usr/lib/ddm/broadcom.py:157
#, python-format
msgid "Used wireless driver: %(drv)s"
msgstr "Použit WiFi ovladač: %(drv)s"

#: usr/lib/ddm/broadcom.py:178
#, python-format
msgid "Broadcom found: %(broadcom)s"
msgstr "Nalezen Broadcom: %(broadcom)s"

#: usr/lib/ddm/broadcom.py:201
#, python-format
msgid "Broadcom chipset not supported or ethernet controller: %(broadcom)s"
msgstr "Čipset Broadcom nebo řadič ethernet není podporován: %(broadcom)s"

#: usr/lib/ddm/broadcom.py:203
#, python-format
msgid "Broadcom chipset not found: %(chip)s"
msgstr "Čipset Broadcom nenalezen: %(chip)s"

#: usr/lib/ddm/broadcom.py:206
#, python-format
msgid "Broadcom pci ID not found: %(id)s"
msgstr "Broadcom pci ID nenalezeno: %(id)s"

#: usr/lib/ddm/broadcom.py:215
#, python-format
msgid "Linux header name to install: %(header)s"
msgstr "Linux název hlavičky k instalaci: %(header)s"

#: usr/lib/ddm/broadcom.py:219
#, python-format
msgid "Download package: %(package)s"
msgstr "Stahnout balíček: %(package)s"

#: usr/lib/ddm/broadcom.py:227
#, python-format
msgid "Download package: %(drv)s"
msgstr "Stahnout balíček: %(drv)s"

#: usr/lib/ddm/broadcom.py:264
msgid "Install downloaded packages"
msgstr "Instalovat stažený balíček"

#: usr/lib/ddm/broadcom.py:272
#, python-format
msgid "Reconfigure package: %(package)s"
msgstr "Překonfigurovat balíček: %(package)s"

#: usr/lib/ddm/broadcom.py:315
#, python-format
msgid "Purge driver: %(drv)s"
msgstr "Odstranit ovladač: %(drv)s"

#: usr/lib/ddm/broadcom.py:325
msgid "Done removing Broadcom drivers"
msgstr "Broadcom ovladače odebrány"

#: usr/lib/ddm/ddm.py:81
msgid "Graphics"
msgstr "Grafiky"

#: usr/lib/ddm/ddm.py:82
msgid "Wireless"
msgstr "WiFi"

#: usr/lib/ddm/ddm.py:83
msgid "Kernel"
msgstr "Kernel"

#: usr/lib/ddm/ddm.py:84
msgid "Hardware"
msgstr "Hardware"

#: usr/lib/ddm/ddm.py:85
msgid "Activated driver"
msgstr "Ovladač aktivován"

#: usr/lib/ddm/ddm.py:86
msgid "Alternative drivers"
msgstr "Alternativní ovladače"

#: usr/lib/ddm/ddm.py:167 usr/lib/ddm/ddm.py:198
msgid "Current drivers are correct"
msgstr "Současné ovladače jsou vpořádku"

#: usr/lib/ddm/ddm.py:208
msgid "PAE already installed"
msgstr "PAE je již nainstalováno"

#: usr/lib/ddm/ddm.py:210
msgid "PAE support only for 32-bit systems"
msgstr "PAE podporuje pouze 32-bitové systémy"

#: usr/lib/ddm/ddm.py:250
msgid "Activate"
msgstr "Aktivace"

#: usr/lib/ddm/ddm.py:250
msgid "Driver"
msgstr "Ovladač"

#: usr/lib/ddm/ddm.py:250
msgid "Version"
msgstr "Verze"

#: usr/lib/ddm/ddm.py:250
msgid "Description"
msgstr "Popis"

#: usr/lib/ddm/ddm.py:265
msgid " (recommended)"
msgstr " (doporučeno)"

#: usr/lib/ddm/ddm.py:310
msgid "Unknown driver"
msgstr "Neznámý ovladač"

#: usr/lib/ddm/ddm.py:313
msgid ""
"Unknown driver found.\n"
"\n"
"Please remove before installing drivers with DDM."
msgstr ""
"Nalezen neznámý ovladač.\n"
"\n"
"Odeberte prosím před instalací ovladačů pomocí DDM."

#: usr/lib/ddm/ddm.py:358
msgid "Driver activation"
msgstr "Aktivace ovladače"

#: usr/lib/ddm/ddm.py:358
#, python-format
msgid ""
"Are you sure you want to install the following driver:\n"
"\n"
"%(drv)s"
msgstr ""
"Jste si jistí, že chcete nainstalovat následující ovladač:\n"
"\n"
"%(drv)s ?"

#: usr/lib/ddm/ddm.py:363 usr/lib/ddm/ddm.py:503
msgid "Locked"
msgstr "Uzamčeno"

#: usr/lib/ddm/ddm.py:363
#, python-format
msgid ""
"Could not get lock on %(file)s.\n"
"\n"
"Close any programs locking the file and try again."
msgstr ""
"Nelze uzamknout %(file)s.\n"
"\n"
"Ukončete programy, které soubor uzamykají a zkuste to znovu."

#: usr/lib/ddm/ddm.py:441
msgid "Driver installed"
msgstr "Ovladač nainstalován"

#: usr/lib/ddm/ddm.py:441
#, python-format
msgid ""
"Driver installed: %(drv)s\n"
"\n"
"Drivers will be used on next boot.\n"
"Do you want to reboot now?"
msgstr ""
"Nainstalován ovladač: %(drv)s\n"
"\n"
"Ovladače budou spuštěny po příštím restartu.\n"
"Chcete restartovat počítač nyní?"

#: usr/lib/ddm/ddm.py:503
#, python-format
msgid ""
"Could not get lock on %(file)s.\n"
"\n"
"Close any programs locking the file and restart DDM."
msgstr ""
"Nelze uzamknout %(file)s.\n"
"\n"
"Ukončete programy, které soubor uzamykají a restartujte DDM."

#: usr/lib/ddm/ddm.py:554
msgid "Checking your hardware..."
msgstr "Prověřování vašeho hardware..."

#: usr/lib/ddm/ddm.py:619
msgid "Start automatic driver install"
msgstr "Start automatické instalace ovladače"

#: usr/lib/ddm/intel.py:47
#, python-format
msgid "Intel card found: %(card)s"
msgstr "Nalezena karta Intel: %(card)s"

#: usr/lib/ddm/intel.py:53
#, python-format
msgid "Intel driver found: %(drv)s (%(status)s)"
msgstr "Nalezen ovladač Intel: %(drv)s (%(status)s)"

#: usr/lib/ddm/main.py:66
#, python-format
msgid "Write debug information to file: %(path)s"
msgstr "Zápis ladících informací do souboru: %(path)s"

#: usr/lib/ddm/main.py:70
#, python-format
msgid "Machine info: %(info)s"
msgstr "Informace o stroji: %(info)s"

#: usr/lib/ddm/main.py:72
#, python-format
msgid "DDM version: %(version)s"
msgstr "Verze DDM: %(version)s"

#: usr/lib/ddm/main.py:77
msgid "apt-listbugs is installed and might interfere with driver installation"
msgstr "apt-listbugs je nainstalován a může kolidovat s instalací ovladače"

#: usr/lib/ddm/main.py:107 usr/lib/ddm/main.py:109
msgid "Please enter your password"
msgstr "Vložte prosím svoje heslo"

#: usr/lib/ddm/main.py:115
msgid "DDM - Kernel"
msgstr "DDM - Linuxové jádro"

#: usr/lib/ddm/main.py:116
msgid ""
"You do not have the latest kernel installed.\n"
"\n"
"Use the Update Manager to update your system."
msgstr ""
"Nemáte nainstalovánu poslední verzi linuxového jádra.\n"
"\n"
"Použijte Update Manager a aktualizujte svůj systém."

#: usr/lib/ddm/main.py:120
msgid "DDM - Live environment"
msgstr "DDM - Živé prostředí"

#: usr/lib/ddm/main.py:121
msgid ""
"DDM cannot run in a live environment\n"
"\n"
"To force start, use the --force argument"
msgstr ""
"DDM nemůže běžet v živém prostředí\n"
"\n"
"Pro vynucení spuštění použijte argument --force"

#: usr/lib/ddm/main.py:125
msgid "DDM - Debian based"
msgstr "DDM - Založeno na Debianu"

#: usr/lib/ddm/main.py:126
msgid ""
"Cannot determine the base distribution (debian or ubuntu)\n"
"\n"
"To force start, use the --force argument"
msgstr ""
"Nelze určit výchozí distribuci (debian nebo ubuntu)\n"
"\n"
"Pro vynucení spuštění použijte argument --force"

#: usr/lib/ddm/main.py:130
msgid "DDM - Internet"
msgstr "DDM - Internet"

#: usr/lib/ddm/main.py:131
msgid ""
"You do not seem to have an internet connection\n"
"\n"
"To force start, use the --force argument"
msgstr ""
"Vypadá to, že nemáte připojení k internetu\n"
"\n"
"Pro vynucení spuštění použijte argument --force"

#: usr/lib/ddm/nvidia.py:56
msgid "Update apt"
msgstr "Aktualizace apt"

#: usr/lib/ddm/nvidia.py:59
msgid "Install nvidia-detect"
msgstr "Instalace nvidia-detect"

#: usr/lib/ddm/nvidia.py:67
msgid "Cannot install nvidia-detect: abort"
msgstr "nvidia-detect nelze nainstalovat: přerušeno"

#: usr/lib/ddm/nvidia.py:91
#, python-format
msgid "Nvidia card found: %(card)s"
msgstr "Nalezena karta Nvidia: %(card)s"

#: usr/lib/ddm/nvidia.py:101
#, python-format
msgid "Nvidia driver found: %(drv)s (%(status)s)"
msgstr "Nalezen ovladač Nvidia: %(drv)s (%(status)s)"

#: usr/lib/ddm/nvidia.py:159
msgid "Removing Nvidia drivers"
msgstr "Odebrání Nvidia ovladačů"

#: usr/lib/ddm/nvidia.py:184
msgid "Nvidia display driver"
msgstr "Nvidia ovladač displeje"

#: usr/lib/ddm/nvidia.py:188
msgid "Nouveau display driver"
msgstr "Nouveau ovladač displeje"

#: usr/lib/ddm/pae.py:42
#, python-format
msgid "Installed PAE: %(pae)s"
msgstr "Nainstalováno PAE: %(pae)s"

#: usr/lib/ddm/pae.py:48
msgid "Multi-core support for 32-bit systems"
msgstr "Podpora více jader pro 32-bitové systémy"

#: usr/lib/ddm/pae.py:58
#, python-format
msgid "Multi-core already installed: %(package)s"
msgstr "Podpora více jader je již nainstalována: %(package)s"

#: usr/lib/ddm/pae.py:59 usr/lib/ddm/pae.py:72
msgid "PAE capable system"
msgstr "Systém podporující PAE"

#: usr/lib/ddm/pae.py:65
msgid "Multi-core system running single-core kernel found"
msgstr "Zjištěn běh jednojádrového linuxového jádra na vícejádrovém systému"

#: usr/lib/ddm/pae.py:69
msgid "PAE not installed"
msgstr "PAE není nainstalováno"

#: usr/lib/ddm/pae.py:78
msgid "PAE kernel cannot be installed: single-core system"
msgstr "PAE linuxové jádro nelze nainstalovat: jednojádrový systém"

#: usr/lib/ddm/pae.py:94
msgid "Done installing PAE"
msgstr "Instalace PAE dokončena"

#: usr/lib/ddm/pae.py:108
#, python-format
msgid "PAE package to remove: %(package)s"
msgstr "Balíček pro odebrání PAE: %(package)s"

#: usr/lib/ddm/pae.py:111
msgid "Done removing PAE"
msgstr "Odinstalování PAE hotovo"

#: usr/lib/ddm/pae.py:113
msgid "Cannot remove PAE when running PAE"
msgstr "PAE nelze odebrat, pokud PAE běží"

#: usr/lib/ddm/via.py:46
#, python-format
msgid "Via card found: %(card)s"
msgstr "Nalezena karta Via: %(card)s"

#: usr/lib/ddm/via.py:52
#, python-format
msgid "Via driver found: %(drv)s (%(status)s)"
msgstr "Nalezen ovladač Via: %(drv)s (%(status)s)"

#: usr/lib/ddm/via.py:103
msgid "Chrome display driver"
msgstr "ovladač displeje"

#: usr/lib/ddm/xorg.py:78
#, python-format
msgid "Used graphics driver: %(module)s"
msgstr "Použit grafický ovladač: %(module)s"

#: usr/lib/ddm/xorg.py:100
msgid "No xorg.conf - create minimal xorg.conf"
msgstr "Chybí xorg.conf - vytvořte minimální xorg.conf"

#: usr/lib/ddm/xorg.py:120
msgid "New xorg.conf written"
msgstr "Nový xorg.conf uložen"

#: usr/lib/ddm/xorg.py:126
msgid "New section appended to xorg.conf"
msgstr "Nová sekce připojena do xorg.conf"
